components/eamxx/src/diagnostics/extraaci_util.hpp

```diff
@@ -0,0 +1,45 @@
+#ifndef EAMXX_EXTRAACI_DIAG_UTIL
+#define EAMXX_EXTRAACI_DIAG_UTIL
+
+#include <map>
+
+namespace scream {
+class ExtraAciDiagUtil {
+ public:
+  std::map<std::string, int> index_map;
+  std::map<std::string, std::string> units_map;
+  int size;
+  std::string pblinvalg;
+
+  ExtraAciDiagUtil() {
+    // Start (post) incrementing size from 0
+    size = 0;
+
+    // pblp
+    index_map["p+"] = size++;
```

Comment:
What are `+` and `^` in the diags names?

Reply:
Are these "common" names in this context? If not, I would rather use some more wordy names...